src/vivarium_census_prl_synth_pop/data/generate_business_names.py

```diff
@@ -23,13 +24,26 @@ def generate_business_names_data(n_total_names: str):
     new_names = pd.Series()
 
     # Generate additional names until desired number of random business names is met
+    cycles = 1
+    logger.info("Initializing random name sampling for business name generator")
     while len(new_names) < n_total_names:
         n_needed = n_total_names - len(new_names)
+        if cycles > 1:
+            logger.info(f"{n_needed} duplicates found.  Generating additional names for the {cycles} time.")
```

Comment:
The logic here had confused me. Consider dropping this if and just logging something like `f"Cycle {cycle}: generating {n_needed} names`

And then log the number of duplicates down when you're already logging how long the dupe check took.